WSL/toc.yml

```diff
@@ -4,18 +4,22 @@
   items:
     - name: What is WSL?
       href: about.md
+    - name: Best practices for set up
```

Comment:
I'm torn. Should this go in overview? Or Tutorials?

To me, I looked for this first in "Tutorials" as I want the tutorial on setting up my dev environment to use WSL.

Reply:
Moved to tutorials :)